DOCS-2336 and DOCS-2369 Synthetics CI/CD Testing Edits (#11447)
DataDog/documentation: content/en/synthetics/api_tests/http_tests.md

---
## Overview

HTTP tests allow you to send HTTP requests to your applications' API endpoints to verify that they are responding to requests and that they meet any conditions you define, such as overall response time, status code, and header or body content.

HTTP tests can run from [managed][1] and [private locations][2] depending on whether you want to monitor your endpoints from outside or inside your network. HTTP tests can run on a schedule, on-demand, or directly within your [CI/CD pipelines][3].

## Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][4] can create, edit, and delete Synthetic HTTP tests. To get create, edit, and delete access to Synthetic HTTP tests, upgrade your user to one of these two [default roles][4].

If you have access to the [custom role feature][5], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

## Configuration

> Now that you have chosen to create an [`HTTP` test][6], define your test's request.

Reviewer: ```suggestion
After choosing to create an [`HTTP` test][6], define your test's request.
```
